bevm/bevm_attr.go

```diff
@@ -0,0 +1,267 @@
+package bevm
```

Comment:
I must say: this code is really well formatted, a pleasure to read!! :)

Reply:
Thanks a lot! :blush: